fenix/app/src/main/res/values/strings.xml

```diff
@@ -1177,6 +1177,15 @@
     <!-- Text shown in the notification that pops up to re-engage the user.
     %1$s is a placeholder that will be replaced by the app name. -->
     <string name="notification_re_engagement_text">Browse with no saved cookies or history in %1$s</string>
+    <!-- Title A shown in the notification that pops up to re-engage the user -->
+    <string name="notification_re_engagement_A_title" tools:ignore="UnusedResources">Browse without a trace</string>
+    <!-- Text A shown in the notification that pops up to re-engage the user.
+    %1$s is a placeholder that will be replaced by the app name. -->
+    <string name="notification_re_engagement_A_text" tools:ignore="UnusedResources">Private browsing in %1$s doesn’t save your info.</string>
```

Comment:
As Nimbus doesn't support substitutions, it'd be useful to have a version of this string with Firefox already embedded in the string.

Reply:
We should really make sure experiments can use placeholders for app name. Please let me know if there is a reason we can't.
I'm also thinking of Focus/Klar: if we have Nimbus experiments on Focus, we need to support these placeholders.
Regardless, we should use what's in the code. We can't scale doubling strings to accommodate all product names